docs/quick_start/quick_start_with_datagrip.md

```diff
@@ -1,41 +1,41 @@
-# 5.Getting Started With Kyuubi and DataGrip
-## 5.1 What is DataGrip
+# Getting Started With Kyuubi and DataGrip
+## What is DataGrip
 DataGrip is a multi-engine database environment released by JetBrains, supporting MySQL and PostgreSQL, Microsoft SQL Server and Oracle, Sybase, DB2, SQLite, HyperSQL, Apache Derby, and H2.
 
 [About DataGrip](https://www.jetbrains.com/datagrip/)
```

Comment:
Can we remove `[About DataGrip]` and just add hyperlink to the first word `DataGrip`?

Reply:
ok， i will resolve